### ISSUE TITLE
Scoping of module prevents include/exclude of top level folders

### ISSUE THREAD

**Issue by Jwan622** [UnitedIncome/serverless-python-requirements#502]
So in a serverless file, I'm trying to install requirements for different lambdas so I'm using serverless-python-requirements and the module and fileName parameters. However I'm also trying to use package include and exclude for files and folders at the same level as the serverless file and not the filepath that the module scopes into. What I notice is that module is limiting the scope to the lambda folder so that top-level shared folders are no longer visible to the serverless file for includes and excludes. How can I install requirements for each lambda while also not limiting the scope of folders and files that I can include into each individual lambda?

In short, say I have files `.serverless`, and folders `lambda1`, `lambda2`, `shared_folder` all on the same level. The `module` for each lambda function is `lambda1` or `lambda2` and then I have fileName: requirements/prod.txt to point to each lambda's requirements.txt file inside each respective folder and that prevents me from a line like `include:shared_folder` from working because of the scoping of the `module` keyword. Any solutions?

**Comment by joh-mue** [comment]
I thing I ran into the same issue (see my [stack overflow post](https://stackoverflow.com/questions/61209464/package-serverless-python-service-with-individually-packaged-functions-but-share)

**Comment by ernesto** [comment]
Same problem here

**Comment by spsoni** [comment]
Same problem for me too.
Ideally, we should be able to add shared code (using include/exclude at global package level)

**Comment by MichaelRBond** [comment]
Ideally, we would be able to specify a requirements filename for each function that needed one different than the default. That way we wouldn't have to worry about the module at all?
